cime_config/cam_autogen.py

```diff
@@ -435,18 +435,23 @@ def generate_physics_suites(build_cache, preproc_defs, host_name,
     if not os.path.exists(physics_blddir):
         os.makedirs(physics_blddir)
     # End if
-    # Collect all source directories
-    atm_phys_src_dir = os.path.join(atm_root, "src", "physics", "ncar_ccpp")
-    source_search = [source_mods_dir, atm_phys_src_dir]
-    # Find all metadata files, organize by scheme name
+    # Set top-level CCPP physics directory
+    atm_phys_top_dir = os.path.join(atm_root, "src", "physics", "ncar_ccpp")
+    # Collect all possible Suite Definition File (SDF) locations
+    atm_suites_path      = os.path.join(atm_phys_top_dir, "suites")
+    atm_test_suites_path = os.path.join(atm_phys_top_dir, "test", "test_suites")
```

Comment:
I'm not sure if there's a mechanism for `cam_autogen` to print an info/warning message, but it might be useful to inform the user that a test SDF is being used (or a confirmation option similar in spirit to `--run-unsupported`)

Reply:
I added a new message that prints to the screen (via `_LOGGER.info`) that lets the user know if test suites are being used.  Let me know if you'd like any modifications to the message or anything.

Reply:
Looks great, thank you!